lib/cn/kblock_builder.c

```diff
@@ -1390,7 +1248,9 @@ kblock_copy_range(
     if (!err && tot_keys > 0) {
         struct blk_list kblk_list;
 
-        err = kbb_finish(kbb, &kblk_list, 0, 0);
+        blk_list_init(&kblk_list);
```

Comment:
Not needed :-)

Reply:
I was getting an uninitialized warning here.

Reply:
Weird...